### ISSUE TITLE
Run pymod in docker container

### ISSUE THREAD

**Issue by dautushenka** [kevinjosethomas/py-mon#25]
Hello everyone,

it looks like that `pymon` requires input stream, therefore it throw a error on running within a container
```
ai-1  | [pymon] watching patterns: *.py
ai-1  | [pymon] enter 'rs' to restart or 'stop' to terminate
ai-1  | [pymon] starting src/main.py
ai-1  | Traceback (most recent call last):
ai-1  |   File "/usr/local/bin/pymon", line 8, in <module>
ai-1  |     sys.exit(main())
ai-1  |              ^^^^^^
ai-1  |   File "/usr/local/lib/python3.11/site-packages/pymon/main.py", line 64, in main
ai-1  |     cmd = input()
ai-1  |           ^^^^^^^
ai-1  | EOFError: EOF when reading a line
```

I suggest add addition flag to disable pymon commands during execution.

Thank you

**Comment by kevinjosethomas** [comment]
Similar issue with #24, I will look into it

**Comment by kevinjosethomas** [comment]
This is now fixed in version ``2.0.5``. Please use the ``-c`` flag to run in clean mode. This will remove all logging from pymon, and disable commands like ``stop`` and ``rs``.
